packages/e2e_tests/cypress/e2e/section-51-advice.cy.js

```diff
@@ -0,0 +1,33 @@
+import { PO_Section51 } from '../pageObject/section-51-advice/PO_Section51';
+
+const section51 = new PO_Section51();
+
+describe('User can review and search Section 51 advice', () => {
+	it('Can navigate to the Section 51 page', () => {
+		cy.clearCookies();
+		cy.visit('/projects/EN010120/');
+		section51.clickSection51Link();
+		cy.url().should('include', '/s51advice');
+		section51.checkMainH1Visible();
+	});
+
+	it('Can change the number of results displays to 50', () => {
+		section51.changeNumberOfResults('50');
+		cy.url().should('include', 'itemsPerPage=50');
+	});
+
+	it('Can perform a search', () => {
+		section51.enterSearchTerm('meeting');
+		cy.get('#search-button').click();
```

Comment:
Can we assert the url has the search term like we did for pagination in the test above